lib/http3/server.c

```diff
@@ -1199,6 +1203,11 @@ static int handle_input_expect_headers(struct st_h2o_http3_server_stream_t *stre
         }
         return 0;
     }
+
+    if (h2o_timeval_is_null(&stream->req.timestamps.request_begin_at)) {
```

Comment:
I'm not sure if we need this `if`. We receive only one request on each stream, and that is guaranteed somewhere else.